jina/serve/consensus/jina_raft/fsm.go

```diff
@@ -45,15 +46,23 @@ func NewExecutorFSM(target string, LogLevel string, raftID string) *executorFSM
 
     conn, _ := executor.newConnection()
     defer conn.Close()
+
     client := pb.NewJinaDiscoverEndpointsRPCClient(conn)
     response, err := client.EndpointDiscovery(context.Background(), &emptypb.Empty{})
     if err != nil {
-        fsm_logger.Error("Error getting endpoints discovery", "error", err)
+        fsm_logger.Error("Error calling endpoints discovery", "error", err)
     }
+
+    // TODO(niebayes): regenerate proto files.
+    read_endpoints := response.ReadEndpoints
```

Comment:
no need for this, the logic of `write` and `read` was there, there is no other set of endpoints